scripts/migrate_all_sll_cert.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+/home/ubuntu/deployment/scripts/convert_ssl.sh /home/ubuntu/secrets/origin_a_com.p12 /home/ubuntu/secrets/nginx_certs/origin_a_com.crt.pem /home/ubuntu/secrets/nginx_certs/origin_a_com.key.pem
```

Comment:
Can we have a bash script that given list of origins will generate all necessary nginx files? I mean `b.com.conf`, `a.com.conf` etc. `origin` can stay as we don't serve certificates for it. If we have to support another host like `c.com` then right now we have to repeat the same process again and again which is a huge hussle.